Godeps/Godeps.json

```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "github.com/letsencrypt/boulder",
 	"GoVersion": "go1.10",
-	"GodepVersion": "v80",
+	"GodepVersion": "v79",
```

Comment:
Looks like you need to update `godep`.

Reply:
Sure, but note it doesn't actually break the build; we have a `sed` in test.sh that removes GodepVersion before comparison, because we don't consider it that important.